### DISCUSSION TITLE
Possible to filter "review in progress" in pull request lists

### DISCUSSION THREAD

**Discussion by brange** [community/community#18639] [Pull Requests]
It would be nice if it would be possible to get a list of pull requests where @me is assigned as reviewer and have started to leave some comments on the pull request but no final answer (accept or change requested).

I'm looking at https://github.com/pulls?q=is%3Aopen+is%3Apr+review-requested%3A%40me+ but I'm missing some pull requests where I have started to review, but not finish the review.

**Comment by BurAndBY** [comment]
Try sorting pull requests by @brange - https://github.com/issues?q=is%3Aopen+is%3Aissue+%brange

**Reply by brange** [reply]
Link doesn't work, but I tried searching with `is:open is:issue @brange` and it did **not** gave me what I wanted.
